app/models/payment_provider_setup.rb

```diff
@@ -9,8 +9,26 @@ module FlexCommerce
   #
   #
   #
-  class PaymentProviderSetup < FlexCommerceApi::ApiBase
-    belongs_to :payment_provider
+  class PaymentProviderSetup
```

Comment:
This can be converted to a module, with `self.call` taking the attributes, no need to initialise an object.